### ISSUE TITLE
[v1.1] graphql transfer record misses `to_account` 

### ISSUE THREAD

**Issue by Keith-CY** [Magickbase/godwoken_explorer#488]
![image](https://user-images.githubusercontent.com/7271329/167304613-29d6b7aa-98f3-465a-9ec0-cb87e858f3b1.png)

tx hash: `0xf419a36c026efb901c5f239c7e0e64d6897dab3a70a97f7345f8589e2055ba74`

The third record of transfers has a `null` to_account

**Comment by Naupio** [comment]
to_address_hash = '0x0000000000000000000000000000000000000000' was not exist in `account` table

**Comment by Naupio** [comment]
Haven't account with address `0x0000000000000000000000000000000000000000`, that's why return `null` of to_account

**Comment by Keith-CY** [comment]
> Haven't account with address `0x0000000000000000000000000000000000000000`, that's why return `null` of to_account

Why not return 
```
{
   "eth_address": "0x0000000000000000000000000000000000000000"
}
```

**Comment by Naupio** [comment]
use to_address for query
change `to_address_hash` name to `to_address`
change `from_address_hash` name to `from_address`

**Comment by Cedar67** [comment]
Regression test result: Resolved in v1.1
Environment: https://api.v1-betanet.gwscan.com/graphql
test data and record:
tx hash: 0xaf0be54aadf00c5717e9380269146ab7e330cc195a2e5c181480aef1a4f552e7
test Result:
<img width="1154" alt="image" src="https://user-images.githubusercontent.com/64718893/169337324-2f03e072-63e4-4ae9-a598-5e91f5ea687e.png">
